.github/workflows/build-and-tests.yml

```diff
@@ -311,6 +311,7 @@ jobs:
     needs:
       - test
       - lint
+      - build
```

Comment:
As test needs build, this should not be needed (but of course does not hurt, either)